pkg/registry/extensions/rest/thirdparty_controller.go

```diff
@@ -126,6 +133,13 @@ func MakeThirdPartyPath(group string) string {
 	return thirdpartyprefix + "/" + group
 }
 
+func getFullThirdPartyPath(group, resource string) string {
+	if len(group) == 0 {
+		return thirdpartyprefix
+	}
+	return thirdpartyprefix + "/" + group + "/" + resource
```

Comment:
@deads2k yes, I am thinking about this PR together with the second issue: `Multiple Versions - it's currently impossible to add more than one version for a group`